src/Microsoft.Data.SqlClient/add-ons/AzureKeyVaultProvider/LocalCache.cs

```diff
@@ -77,9 +77,12 @@ internal TValue GetOrCreate(TKey key, Func<TValue> createItem)
                     AbsoluteExpirationRelativeToNow = TimeToLive
                 };
 
+                AKVEventSource.Log.TryTraceEvent("Encryption key added to local cache.");
```

Comment:
This cache can contain both keys and signature verification results.

Reply:
Below wer're adding key, with a new cache Entry, does this statement need to change?

Reply:
The message says an encryption key was added but this cache is currently used for both encryption keys and signature verification results. I don't think there's an easy way to differentiate between the two.

Reply:
I only meant to say "The **key** for the cache entry" is added to the cache.
Not what it finds as a result of "createItem()". I can reword it better or simply ignore this message then?

Reply:
I think it should remain since caching can be disabled and this event could be helpful in that case. A generic message would be best.